### DISCUSSION TITLE
the Wiggle Jiggle Wobble artifact

### DISCUSSION THREAD

**Discussion by kaelsonofkrypto** [nagadomi/nunif#137] [General]
hi, sorry i don't know how to describe the artifact any other way, but i have noticed sometimes there is flickering , no matter what flicker reduction setting i use, sometimes there is a wiggle on the background behind people that is more than likely caused by flickering, is it possible to look into a better flicker reduction option, and also an auto align option. Thanks for everything

here is a few items that a friend recommended that may work. Maybe could be incorporated in iw3 or maybe something even better than these

Deflicker
http://avisynth.nl/index.php/DeFlicker

Temporal Smoothing
http://avisynth.nl/index.php/TTempSmooth

http://avisynth.nl/index.php/FluxSmooth

**Comment by Ryudoadema** [comment]
Good suggestion! The wiggle jiggle wobble seems like the worst con of zoe any n V2 and the Any V2's in general. It's fairly constant on almost any edges. Flicker reduction doesn't make much of a difference that I can see.
